contracts.js

```diff
@@ -92,6 +98,9 @@ const newDelegatedSending = (web3, address) =>
 const newConditionalStarRelease = (web3, address) =>
   newContract(web3, address, conditionalStarReleaseAbi);
 
+const newMulticall = (web3, address) =>
+  newContract(web3, address, multicallAbi)
```

Comment:
This is... a contract for doing batch reads from other contracts? This seems fancier than it needs to be, can we not generate a batch of RPC calls to send in a single request?

Reply:
>This is... a contract for doing batch reads from other contracts?
Yeah, that's basically what it is.
>can we not generate a batch of RPC calls to send in a single request?
One could. I got into the habit of doing it this way when we were making many rpc calls on a backend for an unrelated project. It was important to reduce the number of calls because we were getting rate limited wrt concurrent requests even on a paid plan. I'm not sure what the default metamask infura rate limit is or what order of magnitude it tolerates. Could be that 16 is fine. I think this way is quicker, but could try with a Promise.all() if you'd prefer

Reply:
Yeah, I'd rather avoid adding the dependency to this library, especially if it still provides the necessary building blocks to let library users do the fancy thing on their end if they really want it.

Reply:
done!